server/lib/nl/fulfillment/types.py

```diff
@@ -159,3 +159,27 @@ class ChartSpec:
   # Dict of sv -> place -> facetid to use
   sv_place_facet_id: Sv2Place2Facet
   info_message: str
+
+
+@dataclass
+class PropertyValue:
```

Comment:
Add comments?  Unclear if this is a triple or just values?  If the former, call it Triple?